<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1051</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="5">
      <widget class="QPushButton" name="cursado">
       <property name="text">
        <string>Cursado</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="checkBox03">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="9">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="botton08">
       <property name="text">
        <string>EMB5039</string>
       </property>
      </widget>
     </item>
     <item row="3" column="10">
      <widget class="QPushButton" name="pushButton_11">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="3" column="11">
      <widget class="QPushButton" name="pushButton_12">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="5" column="9">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="4" column="9">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QPushButton" name="pushButton_13">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="5" column="11">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="4" column="10">
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="5" column="8">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="4" column="11">
      <widget class="QPushButton" name="pushButton_9">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="3" column="8">
      <widget class="QPushButton" name="pushButton_10">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="botton05">
       <property name="text">
        <string>EMB5034</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="pushButton_14">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="4" column="8">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="5" column="10">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="11">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 57 italic 16pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>Controle Curricular ENGENHARIA MECATRÔNICA | PPC 2016/1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_16">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="botton02">
       <property name="text">
        <string>EMB5035</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="pushButton_15">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>60</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="botton04">
       <property name="text">
        <string>EMB5005</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="botton01">
       <property name="text">
        <string>EMB5001</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="botton03">
       <property name="text">
        <string>EMB5683</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="botton07">
       <property name="text">
        <string>EMB5600</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="botton06">
       <property name="text">
        <string>EMB5037</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="botton09">
       <property name="text">
        <string>EMB5007</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1" colspan="3">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Disciplina Selecionada:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>60</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="14" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="checkBox02">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="6" colspan="2">
      <widget class="QPushButton" name="ncursado">
       <property name="text">
        <string>Não Cursado</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1" colspan="11">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1" colspan="11">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="checkBox06">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QCheckBox" name="checkBox07">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QPushButton" name="disponiveis">
       <property name="text">
        <string>Disponiveis</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="botton11">
       <property name="text">
        <string>EMB5006</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QPushButton" name="botton13">
       <property name="text">
        <string>EMB5630</string>
       </property>
      </widget>
     </item>
     <item row="7" column="3">
      <widget class="QPushButton" name="botton12">
       <property name="text">
        <string>EMB5626</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="botton10">
       <property name="text">
        <string>EMB5029</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="checkBox05">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="checkBox04">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBox01">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="checkBox08">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QCheckBox" name="checkBox09">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QCheckBox" name="checkBox10">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QCheckBox" name="checkBox11">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QCheckBox" name="checkBox12">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QCheckBox" name="checkBox13">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1075</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
